examples/nestjs/provider/test/pact/pact-producer-config-options.service.ts

```diff
@@ -0,0 +1,65 @@
+import { Injectable } from '@nestjs/common';
+import { PactProducerOptionsFactory, PactProducerOptions } from 'nestjs-pact';
```

Comment:
Unless I'm missing something, I think these are meant to be named `PactProvider`... not `PactProducer`...  (probably the problem is in nestjs-pact).